server/repository/src/diesel_extensions.rs

```diff
@@ -101,11 +101,17 @@ pub trait OrderByExtensions: Sized {
 
 impl<T> OrderByExtensions for T where T: Expression {}
 
+pub mod date_coalesce {
+    use diesel::sql_types::{Date, Nullable};
+    sql_function! { fn coalesce(x: Nullable<Date>, y: Date) -> Date; }
+}
```

Comment:
Put it into a `mod` to avoid potential name conflicts if we need to implement coalesce() for other types